lib/appsignal/utils.rb

```diff
@@ -14,12 +14,81 @@ def sanitize(params, only_top_level = false, key_sanitizer = nil)
     end
     extend ClassMethods
 
+    def self.data_generate(body)
+      Data.generate(body)
+    end
```

Comment:
I don't think we need to add convenience methods on Appsignal::Utils for every util class we add